### DISCUSSION TITLE
[2021/09/28] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#166] [General]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/issues/164 

 _Created from https://github.com/wantedly/frontend-night/issues/65 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by izumin5210** [comment]
## [Rome will be written in Rust 🦀](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust)

**Reply by kobayang** [reply]
https://qiita.com/rana_kualu/items/dcadf11b3e97db1dded6

日本語ver

**Comment by chloe463** [comment]
## [JSer Info](https://jser.info/2021/09/27/lit-2-gatsby-4-beta-partytownthird-party-script-into-worker/)

**Comment by kobayang** [comment]
https://note.com/mercari_data/n/nb1044a53c1ab
https://netflixtechblog.com/what-is-an-a-b-test-b08cc1b57962

ABテスト周りの話（まだ読んでない）

**Comment by kobayang** [comment]
2020年に立ち上げたWebフロントエンド構成の振り返り
https://zenn.dev/yoshiko/articles/32371c83e68cbe

**Comment by izumin5210** [comment]
[イミュータブルにデータを扱うライブラリと Stage 2 Record & Tuple](https://zenn.dev/petamoriken/articles/f07f48139d9ba1)
[遅延リストを扱うライブラリと Stage 2 Iterator Helpers](https://zenn.dev/petamoriken/articles/062461b79e0d8f)

**Comment by kobayang** [comment]
GraphQL採用サービスで任意カラムを取得できる脆弱性を見つけた話
https://zenn.dev/mipsparc/articles/a818970a19ade6

**Comment by chloe463** [comment]
https://github.com/calendso/calendso

**Comment by prsdta** [comment]
[Making links easier to see and read on GOV.UK](https://designnotes.blog.gov.uk/2021/07/07/making-links-easier-to-see-and-read-on-gov-uk/)

**Reply by prsdta** [reply]
https://design-system.service.gov.uk/patterns/

**Comment by kobayang** [comment]
[Production Ready GraphQL](https://book.productionreadygraphql.com/)

いずみんがこの本いいと言ってたと又聞きした。感想を聞きたい。

**Reply by chloe463** [reply]
@luvtechno さんにおねだりして輪読会を開きたいという話をしてた

**Reply by hayaokimura** [reply]
https://wantedly.slack.com/archives/CA46K091P/p1632824781152800
言ってみた

**Comment by kobayang** [comment]
https://github.com/BuilderIO/partytown

仕組みについて（後で読む） → https://dev.to/adamdbradley/introducing-partytown-run-third-party-scripts-from-a-web-worker-2cnp

**Comment by kobayang** [comment]
React18 で入る予定の機能

https://yagmurcetintas.com/journal/whats-new-in-react-18#automatic-batching

**Reply by kobayang** [reply]
automatic batching は嬉しい。
(setXXX を 2回読んでも 1回の re-render になる）
